<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proyecto de Visi√≥n Artificial</string>
  </property>
  <widget class="QFrame" name="imageFrameS">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="imageFrameD">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Image</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxMerge">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>150</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Corners</string>
   </property>
  </widget>
  <widget class="QFrame" name="imageFrameS2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="imageFrameD2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="initDButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Initialize Disparity</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="propDButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Propagate Disparity</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loadGroundButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Ground Truth</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>350</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Estimated disparity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>420</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>True disparity</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>370</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>440</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
